### DISCUSSION TITLE
如何刷題

### DISCUSSION THREAD

**Discussion by Doreen331** [1101-GS4719-C/Python-Community-GS4719-C#951] [General]
上一篇文分享刷題的重要，但還是很多人不知道要從何開始，甚至沒聽過leetcode這個東西，下面這篇文章有詳細的解釋，適合想要了解leetcode的超級新手

我一開始想用leetcode開始刷題的時候也是超級挫折，只能一直看別人的寫法，但後來發現是自己太菜了，只能趕快逃回家練基本功，大家可以善用leetcode找到學習的動力😢

https://haren.medium.com/leetcode%E5%88%B7%E9%A1%8C%E6%96%B0%E6%89%8B%E5%85%88%E5%82%99%E8%88%87%E7%9B%B8%E9%97%9C%E7%9F%A5%E8%AD%98%E5%88%86%E4%BA%AB-681328997c86

**Comment by Evan0979** [comment]
題之前還是要先把基本功學好,不然很容易就只學會一些片面的東西!!

**Comment by ghost** [comment]
如果有基本知識基底的話，leetcode將可以很有效的讓你融會貫通

**Comment by dinosaur1009** [comment]
leetcode感覺對初學者來說還是有相當難度哇！

**Comment by harrylee1971** [comment]
還蠻需要的，可以把自己不會的面向每一個都可以補齊

**Comment by HsuChen88** [comment]
我之前使用LeetCode時很挫折，像連結中筆者所說，我的方向錯誤，現在已了解自己的弱點也知道怎麼用LeetCode了，是時候重新登入了。

**Comment by orianna901001** [comment]
我在學資料結構的時候，一直都蠻喜歡使用leetcode來刷題目，一方面雖然很挫折但其實能學到很多，許多書面知識在電腦上跑起來，才能知道實際應用到底是怎麼樣，不過我也很久沒有使用leetcode了。

**Comment by RogelioKG** [comment]
基本上就是資結和演算法要很熟再來玩 LeetCode，
如果有初學者的朋友想刷題，結果被 LeetCode 勸退，
可以建議你先刷 [zerojudge](https://zerojudge.tw/Problems)。
接著再刷 [Online Judge](https://onlinejudge.org/index.php?option=com_onlinejudge&Itemid=8&category=1)，
重點就是：要資結和演算法！
所以沒有學的朋友，要補課啦🥹

**Comment by brian431** [comment]
隨著人工智慧寫程式的能力越來越進步，不知道之後的公司面試還會不會以考類似leetcode的題目為主？
